### PR TITLE
Home

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Please review

### CODE REVIEW

**Comment by ashifkhn** [comment]
Looks good. Good to merge.

File: src/components/VideoPlayer/Video/Video.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    }
+  };
+
+  useEffect(() => {
+    if (isAuth) {
+      if (history.some((video) => video._id === id)) setInHistory(true);
+    }
+  }, [isAuth]);
+  return (
+    <video
```
Comment (ivinayakg) [inline src/components/VideoPlayer/Video/Video.tsx:64]: Beautiful.
Kudos to you fro creating your own video player

File: src/components/VideoPlayer/videoActions/BottomBar/SecondaryVideoActions/SecondaryVideoActions.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { BtnFullScreen, BtnMute, BtnNextTrack, BtnPlay, BtnPrevTrack, BtnRepeat } from "components/VideoPlayer/videoActions/buttons";
```
Comment (ivinayakg) [inline src/components/VideoPlayer/videoActions/BottomBar/SecondaryVideoActions/SecondaryVideoActions.tsx:1]: dont try to break things to this level.
Like you could have written this code in single file under the videoActions folder.
You can write multiple components in a single file

File: src/reducers/videos/videosReducer.test.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
+import { VideosStateType } from "__types__/context/videosContext.types"
+import { SetErrorType, SetLoadingTrueType, SetVideosType } from "__types__/reducer/videosReducerActions.types"
+import { vidoesReducer } from "./videosReducer"
+
+describe('test all videos reducer',() =>{
```
Comment (ivinayakg) [inline src/reducers/videos/videosReducer.test.ts:5]: 🔥🔥🔥 for writing tests also

**Review by ivinayakg — APPROVED** [review]
I enjoyed the code but just one correction for the future don't break your components to a very small component

**Comment by HarshnaNHaswani** [comment]
Thank you @ivinayakg, I'll be sure to write smaller components in the same file

**Pull request merged**